airbyte-integrations/connectors/source-twitter/source_twitter/manifest.yaml

```diff
@@ -13,7 +13,11 @@ definitions:
       api_token: "{{ config.api_key }}"
     request_parameters:
       query: "{{ config['query'] }}"
-      tweet.fields: "author_id,conversation_id,created_at,in_reply_to_user_id,lang"
+      tweet.fields: '{{ ",".join(config["tweet_fields"]) }}'
```

Comment:
if the user has old config what next? 


look like braking changes

Reply:
Yes, it is a breaking change. User now has to specify the fields that they want in the config - https://github.com/airbytehq/airbyte/issues/25415#issue-1680084909

Reply:
yes, that's can guarantee backward compatibility

Reply:
yes, that's can guarantee backward compatibility